src/views/Shapes/Circle.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { Modifiers } from "../../utils/modifiers";
+import { Rectangle } from "./Rectangle";
+
+type CircleProps = Omit<Modifiers,"backgroundColor"> & {
+    fill: string
+}
+
+export const Circle: React.FC<CircleProps> = ({
+    fill,
+    opacity,
+    frame,
+    scaleEffect,
+    padding,
+    border,
+    shadow,
+    zIndex,
+    style,
+    onAppear,
+    onDisappear,
+}) => {
+    var width: any = frame?.width;
+    var height: any = frame?.height;
+
+    if (width && height) {
+        try {
+            // width/height could be a string
+            const size = Math.max(width, height);
+            width = size;
+            height = size;
+        } catch (e) {}
+    } else if (width) {
+        height = width;
+    } else if (height) {
+        width = height;
+    }
+
+    return (
+        <Rectangle
+            {...{
+                fill,
+                opacity,
+                frame: { width, height },
+                cornerRadius: Number.MAX_SAFE_INTEGER,
```

Comment:
I think I'd rather use a percentage value here, I think 50% works. But we will need to make the type `number | string`

Reply:
I'm game, but SwiftUI only allows frame and cornerRadius as number. Would you lean towards using theirs or enabling the React Native functionality?